posts/2020-09-03-Planning-2021-Roadmap.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Planning the 2021 Roadmap"
+author: The Rust Core Team
+release: false
+---
+
+The core team is beginning to think about the 2021 Roadmap, and we want to hear from the community. We’re going to be running two parallel efforts over the next several weeks: the 2020 Rust Survey, to be announced next week, and a call for blog posts.
+
+Blog posts can contain anything related to Rust: language features, tooling improvements, organizational changes, ecosystem needs — everything is in scope. We would encourage you to try to identify themes or broad areas into which your suggestions fit in, because these help guide the project as a whole. We are trying something a bit different this year. When you write a post, please submit it into [this google form](https://forms.gle/Hv41uA5qJEY89XRm7) so we don’t miss it! We will try to look at posts not submitted via this form, too, but posts submitted here aren’t going to be missed. Any platform — from blogs to GitHub gists — is fine!
+
+To give you some context for the upcoming year, we established these high-level goals for 2020, and we wanted to take a look back at the first part of the year. We’ve made some excellent progress!
+
+- Prepare for a possible Rust 2021 Edition
+- Follow-through on in-progress designs and efforts
+- Improve project functioning and governance
+
+## Prepare for a possible Rust 2021 Edition
+
+There is now an [open RFC](https://github.com/rust-lang/rfcs/pull/2966) proposing a plan for the 2021 edition! There has been quite a bit of discussion, but we hope to have it merged within the next 6 weeks. The plan is for the new edition to be much smaller in scope than Rust 2018. It it is expected to include a few minor tweaks to improve language usability, along with the promotion of various style lints so that they will be “deny by default”. We believe that we are on track for being able to produce an edition in 2021.
```

Comment:
Knowing nothing about the lints, 'the promotion of various style lints so that they will be “deny by default”' sounds really unfriendly, like it's talking about `deny(non_snake_case)` or something. Are these actually style lints?

Reply:
@nikomatsakis wrote this language, I think, but I suspect it's talking about things like omitting `dyn` in `dyn Trait`. I might reword to say something like "edition idiom lints"